project_agile/views/project_agile.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+-->
+<odoo>
+    <template id="pa_web_assets_backend" name="project_agile web assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/project_agile/static/src/js/project_workflow.js" />
+        </xpath>
+    </template>
+
+    <!-- Portal -->
+    <template id="assets_frontend" inherit_id="website.assets_frontend" name="Project portal task edit frontend assets">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/project_agile/static/src/js/portal_extensions.js"></script>
+        </xpath>
+    </template>
+
+
+    <template id="layout" name="Main layout" inherit_id="website.layout">
+        <xpath expr="//t[@t-call-assets='web_editor.summernote' and @t-js='false']" position="replace">
+            <t t-call-assets="web_editor.summernote" t-js="false" groups="website.group_website_publisher,project_agile.group_project_portal_task_edit"/>
+        </xpath>
+        <xpath expr="//t[@t-call-assets='web_editor.summernote' and @t-css='false']" position="replace">
+            <t t-call-assets="web_editor.summernote" t-css="false" groups="website.group_website_publisher,project_agile.group_project_portal_task_edit"/>
+        </xpath>
+        <xpath expr="//t[@t-call-assets='web_editor.assets_editor' and @t-js='false']" position="replace">
+            <t t-call-assets="web_editor.assets_editor" t-js="false" groups="website.group_website_publisher,project_agile.group_project_portal_task_edit"/>
+        </xpath>
+        <xpath expr="//t[@t-call-assets='web_editor.assets_editor' and @t-css='false']" position="replace">
+            <t t-call-assets="web_editor.assets_editor" t-css="false" groups="website.group_website_publisher,project_agile.group_project_portal_task_edit"/>
+        </xpath>
+    </template>
+
+    <template id="portal_my_tasks_task_edit" inherit_id="project.portal_my_tasks" name="portal_my_tasks_task_edit">
+        <xpath expr="//h3" position="inside">
+            <button name='new_task' data-toggle="modal"
+                    data-target=".modal_new_task" class="btn btn-default pull-right mr8"
+                    groups="project_agile.group_project_portal_task_edit">
+                <i class="fa fa-plus fa-lg"/>
+            </button>
+        </xpath>
+
+        <xpath expr="//div[@t-if='tasks']//thead//th[hasclass('col-md-10')]" position="after">
```

Comment:
This has been changed upstream: https://github.com/odoo/odoo/commit/dea07fc29116c36c8a3773eec0fce526a972ba08#diff-2a064778276c73499a520e03c6c99f7fL138

Reply:
Hi @lreficent, Yes, I'm aware of this. For now, I'll remove everything related to portal.